runtime/common/computationkind.go

```diff
@@ -133,7 +133,7 @@ const (
 	//
 	ComputationKindSTDLIBPanic
 	ComputationKindSTDLIBAssert
-	ComputationKindSTDLIBUnsafeRandom
+	ComputationKindSTDLIBRevertibleRandom
```

Comment:
Not sure if I should keep `ComputationKindSTDLIBUnsafeRandom` here, knowing that `unsafeRandom` will be deprecated.

Reply:
I think that's OK, on the FVM side we can probably just use the weights for the old kind for the new one, right @onflow/flow-execution-and-verification ?